### ISSUE TITLE
Show embeddings w umap

### ISSUE THREAD

**Issue by rmattson1008** [rmattson1008/ornet#28]
Followup to #23. I think it will be able to handle this dataset very well. But are you really understanding the embedding space correctly? What is local structure like in higher (but not super high) R^n? It does not seem like each dim in a vector carries the information split equally, it seems like things that "look" like neighbors may be less related than one would think. Honestly I want to just do PCA too. Find "axes of info" and see if it lines up w any bio interpretation.

**Comment by magsol** [comment]
Yeah I think a constellation of viz approaches is going to be the best here: UMAP as one, t-SNE as another, PCA as another.

**Comment by Micky774** [comment]
> Yeah I think a constellation of viz approaches is going to be the best here: UMAP as one, t-SNE as another, PCA as another.

I would actually recommend against t-SNE since it is difficult to tune and often redundant-at-best or inferior to UMAP. Also you may want to consider using UMAP + HDBSCAN to generate a fully-unsupervised density-based hierarchical clustering of the manifold-projected data.

**Comment by magsol** [comment]
> Also you may want to consider using UMAP + HDBSCAN

+1 to this.
